stitchee/concatenate.py

```diff
@@ -148,7 +247,7 @@ def _create_concat_function(concat_method: str, concat_dim: str, concat_kwargs:
 
     # Create appropriate function
     if concat_method == "xarray-concat":
-        return partial(xr.concat, dim=concat_dim, **base_kwargs)
+        return partial(composite_concat, concat_dim=concat_dim, concat_kwargs=concat_kwargs)
```

Comment:
Matching the function name I suggested above:
```suggestion
        return partial(concat_datasets, concat_dim=concat_dim, concat_kwargs=concat_kwargs)
```